docs/resources/stream_condition.md

```diff
@@ -6,6 +6,8 @@ description: |-
 
 ---
 
+# NOTE: this resource will be deprecated in v2+. Use `lightstep_alert` instead.
```

Comment:
while in here marking things for future deprecation, is `metric_condition` another candidate to be deprecated?

Reply:
`metric_condition` already has a deprecation note on it :)